documentation-site/pages/blog/base-web-v7/index.mdx

```diff
@@ -93,7 +93,7 @@ jscodeshift -t node_modules/baseui/codemods/styled-flowfixme.js <transform-path>
 
 Use the -d option for a dry-run and use -p to print the output for comparison.
 
-### $ref -> ref
+### \$ref -> ref
```

Comment:
is this a typo?

Reply:
this was my editor. happy to revert if you'd like

Reply:
it's fine as is - just wasn't sure if it's on purpose